server/package.json

```diff
@@ -15,7 +15,7 @@
     "url": "https://github.com/bash-lsp/bash-language-server"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=11.0.0"
```

Comment:
@skovhus You should probably bump this up to `>=12.0.0`.

Reply:
Thanks for the careful review. But technically we do support node 11.